helm/polaris/README.md.gotmpl

```diff
@@ -89,22 +71,7 @@ eval $(minikube -p minikube docker-env)
     -Dquarkus.container-image.build=true
 ```
 
-### Installing the chart locally
-
-The below instructions assume a local Kubernetes cluster is running and Helm is installed.
-
-#### Common setup
-
-Create the target namespace:
-
-```bash
-kubectl create namespace polaris
-```
-
-Create all the required resources in the `polaris` namespace. This usually includes a Postgres
```

Comment:
Why remove this paragraph? I think it's still useful. We could instead just remove the reference to persistence.xml and replace it with a reference to the jdbc-persistence and the storage secrets.